### PR TITLE
Update contribution link

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Adds tracking parameters to the "Support the Guardian" link in the TIF series page.

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[Add tracking to contributions link in TIF series page](https://trello.com/c/9wKueW7f/609-add-tracking-to-contributions-link-in-tif-series-page)

File: static/src/javascripts/projects/journalism/views/audioSeriesContributions.html [hunk 1]
```diff
@@ -8,7 +8,7 @@ <h2>Support The Guardian</h2>
                     <br> journalism the world needs</p>
                 <a class="contributions__contribute contributions__option-button"
                    data-link-name="audio-series-contributions"
-                   href="<%= supportContributeURL %>">
+                   href="<%= supportContributeURL %>?acquisitionData=%7B%22componentType%22%3A%22ACQUISITIONS_OTHER%22%2C%22source%22%3A%22GUARDIAN_WEB%22%2C%22campaignCode%22%3A%22today_in_focus%22%2C%22componentId%22%3A%22audio_series_page%22%7D&INTCMP=today_in_focus">
```
Comment (JustinPinner) [inline static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: Do the `componentType`, `campaignCode` and `INTCMP` have to be agreed beforehand? Maybe they already have, in which case ignore this!

Comment (regiskuckaertz) [reply static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: Yes, they've been given to us by the RR team 🤗

**Comment by ajwl** [comment]
👍

**Pull request merged**